Automattic/jetpack
# Customizer > Custom CSS: Add support for CSS variables

This update modifies the CSSTidy script following the example of #19935

Fixes #19669

#### Changes proposed in this Pull Request:
Adds support for CSS variables in Custom CSS

#### Jetpack product discussion
p1623934752160800-slack-calypso

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Open the Customizer and try to add CSS variables in the Additional CSS.
2. Save
3. Make sure the CSS variables have been saved (not stripped) and they apply to the frontend
